.rhdh/scripts/prepare-restricted-environment.sh

```diff
@@ -964,13 +972,13 @@ EOF
 
     if [[ -n "${TO_DIR}" ]]; then
       "${OC_MIRROR_PATH}" \
-        --config="${TMPDIR}/imageset-config.yaml" \
+        -c "${TMPDIR}/imageset-config.yaml" \
```

Comment:
```suggestion
        --config="${TMPDIR}/imageset-config.yaml" \
```

`--config` still works with v2, no?